cloud/shared/bin/lib/check_test.py

```diff
@@ -3,16 +3,17 @@
 from check import Check
 
 class TestCheck(unittest.TestCase):
-    def test_version_greater_than(self):
+    def test_version_at_least(self):
         check = Check()
 
-        self.assertTrue(check.version_greater_than("1", "2"))
-        self.assertFalse(check.version_greater_than("2", "1"))
-        self.assertTrue(check.version_greater_than("1.1", "1.1"))
-        self.assertTrue(check.version_greater_than("1.1", "1.2"))
-        self.assertTrue(check.version_greater_than("1.1", "1.1.3"))
-        self.assertFalse(check.version_greater_than("1.2", "1.1.3"))
-        self.assertFalse(check.version_greater_than("1.0.4", "1.1.3"))
+        self.assertTrue(check.version_at_least("1", "2"))
+        self.assertFalse(check.version_at_least("2", "1"))
+        self.assertTrue(check.version_at_least("1.1", "1.1"))
+        self.assertTrue(check.version_at_least("1.1", "1.2"))
+        self.assertTrue(check.version_at_least("1.1", "1.1.3"))
+        self.assertFalse(check.version_at_least("1.2", "1.1.3"))
+        self.assertTrue(check.version_at_least("1.0.4", "1.1.3"))
```

Comment:
A bit hard to tell from the GitHub diffing, but this line I switched from "assertFalse" to "assertTrue".